src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -58,7 +58,9 @@
 
 def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
                         kube_config=None, kube_context=None, no_wait=False, tags=None, distribution='auto', infrastructure='auto',
-                        disable_auto_upgrade=False, cl_oid=None, onboarding_timeout="300"):
+                        auto_upgrade=None, cl_oid=None, onboarding_timeout="300", container_registry_repository=None, container_registry_username=None,
+                        container_registry_password=None, arc_agent_version=None):
```

Comment:
'arc_agent_version' param is relevant only for pcr scenarios, so i think it would be better to prefix it with 'container_registry' .

Reply:
ack

Reply:
And what would be the CLI experience if a customer passes the parameter agent_version in case of non pcr scenario? Error out or ignore with warning?

Reply:
going with ignore with warning